Scripts/DCS-BIOS/lib/modules/aircraft_modules/Ka-50.lua

```diff
@@ -0,0 +1,917 @@
+--HUD
+Ka_50:define8BitFloat("HUD_FILTER_POS", 509, { 0, 1 }, "HUD", "Filter position")
+Ka_50:define8BitFloat("HUD_FILTER_HANDLE_POS", 510, { 0, 1 }, "HUD", "Filter lever position")
+Ka_50:definePotentiometer("HUD_BRIGHTNESS", 7, 3001, 8, { 0, 1 }, "HUD", "Brightness Knob")
+Ka_50:define3PosTumb("HUD_MODE", 7, 3002, 9, "HUD", "Reticle/Night/Day mode selector")
+Ka_50:definePushButton("HUD_TEST", 7, 3003, 7, "HUD", "Self-test button")
+Ka_50:defineToggleSwitch("HUD_FILTER_HANDLE", 7, 3004, 510, "HUD", "Filter lever Switch")
+--PUI-800 Weapon Status and Control Panel
+Ka_50:defineToggleSwitch("WEAPONS_MASTER_ARM", 12, 3001, 387, "Weapons Control Panel", "Master arm Switch")
+Ka_50:definePushButton("WEAPONS_JETTISON_AA_MIS", 12, 3002, 401, "Weapons Control Panel", "Emergency jettison of AA missiles button (no function)")
+Ka_50:definePushButton("WEAPONS_JETTISON_EXT", 12, 3003, 402, "Weapons Control Panel", "External stores jettison button (except Vikhr)")
+Ka_50:defineToggleSwitch("WEAPONS_JETTISON_ARM_MODE", 12, 3022, 396, "Weapons Control Panel", "Jettison arm mode - arm fuse/disarm fuse Switch")
+Ka_50:defineToggleSwitch("WEAPONS_MANUAL_AUTO", 12, 3005, 403, "Weapons Control Panel", "Manual/Auto weapon system control Switch")
+Ka_50:defineToggleSwitch("WEAPONS_CANNON_ROUND", 12, 3006, 399, "Weapons Control Panel", "Cannon round selector Switch HE/AP")
+Ka_50:defineMultipositionSwitch("WEAPONS_CANNON_BURST", 12, 3004, 400, 3, 0.1, "Weapons Control Panel", "Long-medium-short cannon burst mode Switch")
+Ka_50:defineToggleSwitch("WEAPONS_CANNON_RATE", 12, 3020, 398, "Weapons Control Panel", "Cannon rate of fire setting Switch")
+Ka_50:defineToggleSwitch("WEAPONS_EXPEDITE_ATGM_JETTISON", 12, 3021, 397, "Weapons Control Panel", "Expedite emergency ATGM launch Switch")
+Ka_50:defineIndicatorLight("WEAPONS_PRESENT_STATION_1", 392, "Weapons Control Panel", "Weapon is present on station 1 light (yellow)")
+Ka_50:defineIndicatorLight("WEAPONS_PRESENT_STATION_2", 393, "Weapons Control Panel", "Weapon is present on station 2 light (yellow)")
+Ka_50:defineIndicatorLight("WEAPONS_PRESENT_STATION_3", 394, "Weapons Control Panel", "Weapon is present on station 3 light (yellow)")
+Ka_50:defineIndicatorLight("WEAPONS_PRESENT_STATION_4", 395, "Weapons Control Panel", "Weapon is present on station 4 light (yellow)")
+Ka_50:defineIndicatorLight("WEAPONS_RDY_STATION_1", 388, "Weapons Control Panel", "Weapon is ready to fire on station 1 light (green)")
+Ka_50:defineIndicatorLight("WEAPONS_RDY_STATION_2", 389, "Weapons Control Panel", "Weapon is ready to fire on station 2 light (green)")
+Ka_50:defineIndicatorLight("WEAPONS_RDY_STATION_3", 390, "Weapons Control Panel", "Weapon is ready to fire on station 3 light (green)")
+Ka_50:defineIndicatorLight("WEAPONS_RDY_STATION_4", 391, "Weapons Control Panel", "Weapon is ready to fire on station 4 light (green)")
+local indPUI800 = nil
+
+Ka_50:addExportHook(function()
+	indPUI800 = Module.parse_indication(6)
```

Comment:
Is this used anywhere other than the below function? If not, perhaps we should move. This into the function so the variable can be kept in the same closure?

Reply:
This I don't understand.